ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -49,4 +50,7 @@ platform:
     username: ${vsphere_user}
     apiVIP: "${vips[0]}"
     ingressVIP: "${vips[1]}"
+networking:
+  machineNetwork:
+    cidr: "${machine_cidr}"
```

Comment:
`.networking.machineNetwork` is a list. You need a `-` for each entry in the list.
```suggestion
  - cidr: "${machine_cidr}"
```